css-layout-classes/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>CSS-Layout-Classes</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <h1>Choose your character!</h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-third">
+        <img src="images/link.png" alt="link">
+      </div>
+      <div class="column-one-third">
+        <h2 class="h2"><strong>Super Smash Bros. Ultimate</strong></h2>
```

Comment:
h2 elements are bold by default, so you don't need a strong tag here, unless it is for accessibility concerns